app/main/plugins/core/basic-apps/linux.js

```diff
@@ -103,7 +103,10 @@ const findIcon = (icon) => {
   )).find(fs.existsSync)
 }
 
-export const toString = ({ name }) => `${name} ${getAbbr(name)}`
+export const toString = ({ name, exec }) => {
+  const binaryName = exec.split('/').pop().split(' ').shift()
```

Comment:
According to https://github.com/nazar-pc/cerebro/blob/ae0d8a6d81b09c7723107fb74551e8c8b2e6376e/app/main/plugins/core/basic-apps/linux.js#L76-L76 it shouldn't be the case, should always be a string.